content/applications/websites/ecommerce/managing_products/catalog.rst

```diff
@@ -0,0 +1,148 @@
+============================
+Organize the product catalog
+============================
+
+The eCommerce catalog is the equivalent of your physical store shelves: it allows customers to see
+what you have to offer. Clear categories, available options, sorting, and navigation threads help
+you structure it efficiently.
+
+Categorize the product catalog
+==============================
+
+In Odoo, there is a **specific category model** for your eCommerce. Using eCommerce categories for
+your products allows you to add a navigation menu on your eCommerce page. Visitors can then use it
+to view all products under the category they select. To do so, go to :menuselection:`eCommerce -->
+Products --> (Your product) --> Sales` and select the :guilabel:`Categories` you want under
+:guilabel:`eCommerce Shop`.
```

Comment:
I would replace it by something like:
Go to :menuselection:`Website app --> eCommerce`, select a product, go to the :guilabel:`Sales` tab and Sales and under the eCommerce Shop, add the categories you want.